src/storhaug

```diff
@@ -399,6 +402,7 @@ setup_create_resources()
             config="${HA_NFS_CONF}" \
             state_fs="${HA_NFS_TYPE}" \
             state_mnt="${HA_MNT_DIR}/${HA_NFS_MNT_DIR}" \
+        op monitor on-fail="block" \
```

Comment:
hmmm...actually, it is hard to say, just a case in my mind, not encountered in real environment.
I think we may control this behavior by setting the "default times of restarts" to 0 to achieve the same behavior(if it is really desired),  not neccessary to force it here.
I will drop this line in the next round, thanks.